File: app/models/sede.py
from typing import List, TYPE_CHECKING
from sqlmodel import SQLModel, Field, Relationship

if TYPE_CHECKING:
    from app.models.departamento import Departamento
    from app.models.empleado import Empleado

class Sede(SQLModel, table=True):
    id: int = Field(default=None, primary_key=True)
    ciudad: str
    direccion: str

    empleados: List["Empleado"] = Relationship(back_populates="sede")

File: Models/Sede.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GestionEmpleadosMaui.Models
{
    public class Sede
    {
        public int Id { get; set; }
        public string Ciudad { get; set; }
        public string Direccion { get; set; }
        
        List<Empleado> Empleados { get; set; }
    }
}
